plugins/TodayAtMun/__init__.py

```diff
@@ -0,0 +1,53 @@
+import discord
+from discord.ext import commands
+
+from Bot import bot
+from Plugin import AutomataPlugin
+from plugins.TodayAtMun.diary_parser import diary_parser
+from plugins.TodayAtMun.Today import Today
+
+
+class TodayAtMun(AutomataPlugin):
+    def __init__(self, manifest, bot):
+        super().__init__(manifest, bot)
+        self.Tod = Today()
+
+    @commands.group()
+    async def today(self, ctx):
+        """ Provides Brief Info Of Significant Dates on the Mun Calendar"""
+        if not ctx.invoked_subcommand:
+            await ctx.send("Please Provide Subcommand e.g !today [soon]")
+
+    @today.command(name="next")
+    async def today_next(self, ctx):
+        # Sends quick update on Muns Next Calendar Date on the Uni Diary
+        self.Tod.set_current_date()
+        self.Tod.findEvent(self.Tod.date)
+        embed = discord.Embed(
+            title=f"{self.Tod.format_date(self.Tod.date)}",
+            description=f"```{self.Tod.infoDay}``` ( !today later ) to get next event",
+            url="https://www.mun.ca/regoff/calendar/sectionNo=GENINFO-0086",
```

Comment:
You're using this string in this file and in your file in which you parse the data from the website, maybe consolidate it to a variable defined in one place and import it into one or the other?

Reply:
```py
        embed = discord.Embed(
            title=f"{self.tod.fdate}",
            description=f"```{self.tod.infoDay}``` ( !today later ) to get next event",
            url="https://www.mun.ca/regoff/calendar/sectionNo=GENINFO-0086",
            colour=discord.Colour.orange(),
        )
```
More like this?

Reply:
As in: `https://www.mun.ca/regoff/calendar/sectionNo=GENINFO-0086` should only be defined once in your codebase, so when you pull the data down from MUN / display the embed, you use the same piece of data :)